utils/check_repo.py

```diff
@@ -1168,27 +1156,27 @@ def check_deprecated_constant_is_up_to_date():
 
 
 def check_repo_quality():
-    """Check all models are properly tested and documented."""
-    print("Checking all models are included.")
+    """Check all models are tested and documented."""
+    print("    - checking all models are included.")
```

Comment:
nit: with the addition of spacing, we can clearly see that these prints belong to `check_repo.py` in `make fixup` 🤗 
 
<img width="666" alt="Screenshot 2024-07-30 at 11 48 10" src="https://github.com/user-attachments/assets/1f65cd25-9fe8-4359-9bd9-ad036d85fc1d">